lib/services/stash-before.js

```diff
@@ -24,12 +23,10 @@ module.exports = function (prop) {
     };
 
     params.query = params.query || {};
-    params.query.$disableStashBefore = true;
+    params.disableStashBefore = true;
 
     return context.service.get(context.id, params)
```

Comment:
Here is a good example where a small change (not mutating existing objects) can make a big difference when it comes to avoiding subtle bugs. Instead of modifying `params` we could probably just do:

```js
return context.service.get(context.id, {
  ...params,
  query: params.query || {},
  disableStashBefore: true
})
```

Reply:
I've updated the PR.